src/components/StyledProgress/StyledProgress.tsx

```diff
@@ -77,6 +89,47 @@ const StyledProgress = ({
           />
         </>
       )}
+      <Box className={classes.belowDetails}>
+        {receivedBelow && (
+          <Tooltip
+            title={t('Received Below Goal')}
+            arrow
+            PopperProps={{
```

Comment:
Here's those same `PopperProps` again. Maybe it would be worth making a `ProgressTooltip` that can be used here and in the other component.